File: src/main/java/me/flegacy/flms/ui/elements/FillerElement.java
package me.flegacy.flms.ui.elements;

import org.bukkit.inventory.ItemStack;

public class FillerElement implements DisplayElement {

    private final ItemStack display;

    public FillerElement(ItemStack display) {
        this.display = display;
    }

    @Override
    public ItemStack getDisplayItem() {
        return display;
    }
}

File: src/main/kotlin/io/github/flegacy/flms/ui/element/FillerElement.kt
package io.github.flegacy.flms.ui.element

import io.github.flegacy.flms.FLMS
import io.github.flegacy.flms.items.ItemLibrary
import org.bukkit.inventory.ItemStack

class FillerElement: InterfaceElement {

    override fun getDisplay(): ItemStack {
        return ItemLibrary.FILLER
    }

}

